### PR TITLE
Create dark mode design token set

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #88 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Separate the light and dark tokens into two directories
* Configure build to generate new token files for dark theme

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: scripts/config-dark.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "include": ["./src/asset/*.json", "./src/size/*.json", "./src/*.json"],
+  "source": ["./src/color-dark/*.json"],
+  "platforms": {
+    "SCSSDarkVariables": {
+      "transformGroup": "scss",
```
Comment (blackfalcon) [inline scripts/config-dark.json:6]: We will need a transform group for standard CSS as well.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: scripts/config-dark.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {
+          "destination": "SCSSDarkVariablesMapFlat.scss",
+          "format": "custom/scss/map-flat",
+          "mapName": "auroTokens",
+          "filter": {
+            "public": true
+          },
+          "options": {
+            "outputReferences": true,
+            "selector": ".dark"
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply scripts/config-dark.json:31]: This was for experimentation purposes based on what I found from the multi-file method. We do not need to and I am removing it in my next patch.

File: scripts/config-dark.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "include": ["./src/asset/*.json", "./src/size/*.json", "./src/*.json"],
+  "source": ["./src/color-dark/*.json"],
+  "platforms": {
+    "SCSSDarkVariables": {
+      "transformGroup": "scss",
+      "prefix": "auro",
+      "comment": "File type: SCSS; variable type: Sass; filter: public",
```
[the remark this answers is not in the document]
Comment (jordanjones243) [reply scripts/config-dark.json:8]: Are you referring to adding `dark mode` into the comment section? Or as another string of data like `darkMode: true`?

[the remark this answers is not in the document]
Comment (blackfalcon) [reply scripts/config-dark.json:8]: Add a reference to `dark mode` in this comments string.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: scripts/config-dark.json (hunk 1 above)
Comment (jordanjones243) [reply scripts/config-dark.json:6]: What exactly do you mean by this? Do you want me to change this to `transformGroup: css`? I copied this code over exactly from the original `config.json` and in my latest patch, added some more `platform` groups that include a transform group for css. Let me know if that was what you were looking for or what I may be missing.

Comment (blackfalcon) [reply scripts/config-dark.json:6]: There should be a CSS output also. More and more teams are moving to using the raw CSS from the CDN versus having to process Sass locally.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**